### PR TITLE
fix(config): support Comet parameters properly

### PR DESCRIPTION
**Pull request by Mart124**
Hello,

This PR fixes / enhances the Eurotronic Comet Z-Wave in various ways.

### 1. Manufacturer specific mode (1st commit)

I have Eurotronic CometZ-Wave devices, which report like this :
https://pastebin.com/raw/zbF6cpfY

As for the Eurotronic Spirit devices, the _Manufacturer mode_ is [supported / documented](https://eurotronic.org/wp-content/uploads/2024/06/bedienungsanleitung-comet-z-wave-en.pdf), but not reported by the device.

```
        "4-64-0-mode": {
          "endpoint": 0,
          "commandClass": 64,
          "commandClassName": "Thermostat Mode",
          "property": "mode",
          "propertyName": "mode",
          "ccVersion": 3,
          "metadata": {
            "type": "number",
            "readable": true,
            "writeable": true,
            "label": "Thermostat mode",
            "min": 0,
            "max": 255,
            "states": {
              "0": "Off",
              "1": "Heat",
              "11": "Energy heat",
              "15": "Full power"
            },
            "stateful": true,
            "secret": false
          },
          "value": 1
        },
```

Interestingly enough is the _Full power_ mode, reported by the device, but not documented by Eurotronic.
Device's firmware may have been updated since the documentation...

Let's then apply the same modification as for the Spirit device :
https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x0148/spirit.json

Result :

image

Note that when enabling the _Manufacturer specific_ mode in Home Assistant through its default / builtin thermostat, it works, but also reports the following warning :
_Failed to call service climate/set_hvac_mode. expected HVACMode or one of 'off', 'heat', 'cool', 'heat_cool', 'auto', 'dry', 'fan_only' for dictionary value @ data['hvac_mode']_

When setting the _Manufacturer specific_ mode manually (through Home Assistant Developer Tools / zwave_js.set_value), the warning does not trigger. Should then be related to the default / builtin thermostat itself.

### 2. Supervision (2nd commit)

There's a report [here](https://github.com/home-assistant/core/issues/103254) indicating that parameters can't be set.
As found on other devices, seems like Comet device does not like supervision. Let's then disable it.
It then fixes https://github.com/home-assistant/core/issues/103254.
**Should a report be made to Eurotronic about supervision ?**

### 3. Valve opening (3rd commit)

Some devices report valve opening parameter, some don't, may be related to firmware version (or to template used, I'm not sure here...).
But even when not initially reported by the device, it is supported.
Let's then enable this useful option.

Result (enabled after 6:00) :

image

Thank you very much 👍

### CODE REVIEW

**Comment by Mart124** [comment]
When enabling the _Manufacturer specific_ mode in Home Assistant through its default thermostat, it works, but also reports the following warning :
_Failed to call service climate/set_hvac_mode. expected HVACMode or one of 'off', 'heat', 'cool', 'heat_cool', 'auto', 'dry', 'fan_only' for dictionary value @ data['hvac_mode']_

When setting the _Manufacturer specific_ mode manually (through Home Assistant Developer Tools / zwave_js.set_value), the warning does not trigger. Should then be related to the default thermostat itself.

**Comment by Mart124** [comment]
@AlCalzone would really be glad to see this merged into next release, if possible of course 😃
Thank you very much 👍

**Comment by gnipp** [comment]
Please tell me, how I may test it. I would like to add a review.
(Path to a descriptions is sufficient - I simply do not know, how to put the code on my Home Assistant instance.)
Thank you.

**Comment by Mart124** [comment]
Unfortunately there's no "easy" solution.
One could be to migrate from Z-Wave JS to Z-Wave JS UI and to create your custom / own template file based on this PR.

**Comment by AlCalzone** [comment]
I'll review when I find the time --> https://github.com/zwave-js/node-zwave-js/issues/6521

**Comment by gnipp** [comment]
I hope, your maintenance issue #6521 was successful so far. There is no better adventure than that. All the best for you and your family!
My I however dare to ask if I can support somehow?

File: packages/config/config/devices/0x0148/cometz_700.json [hunk 1]
```diff
@@ -80,6 +80,17 @@
 				}
 			]
 		},
+		{
+			// Some devices report this param, some don't, may be related to firmware version.
+			// But even when not reported by the device, it is supported.
+			"#": "6",
+			"label": "Valve Opening Percentage Report",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0148/cometz_700.json]: Is this a threshold, meaning if this is set to 5%, the device reports every change over 5%?

File: packages/config/config/devices/0x0148/cometz_700.json [hunk 2]
```diff
@@ -80,6 +80,17 @@
...
+			"valueSize": 1,
+			"unit": "%",
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 0
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0148/cometz_700.json]: Does 0 mean **Disabled**?

**Review by AlCalzone — COMMENTED** [review]
@Mart124 thanks for the PR. I hope you're still around, because I have two small questions.

**Comment by AlCalzone** [comment]
Ok from what I've gathered, my assumptions are correct.

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
Thanks, and thanks for the patience!

File: packages/config/config/devices/0x0148/cometz_700.json (hunk 1 above)
Comment (Mart124) [reply packages/config/config/devices/0x0148/cometz_700.json]: Yes

File: packages/config/config/devices/0x0148/cometz_700.json (hunk 2 above)
Comment (Mart124) [reply packages/config/config/devices/0x0148/cometz_700.json]: and yes 👍

**Comment by Mart124** [comment]
Many thanks @AlCalzone for your review 👍

**Pull request merged**